src/Microsoft.Azure.WebJobs.Host/Config/FluentBindingRule.cs

```diff
@@ -24,7 +26,7 @@ public class FluentBindingRule<TAttribute> : FluentConverterRules<TAttribute, Fl
     {
         private readonly JobHostConfiguration _parent;
 
-        private List<IBindingProvider> _binders = new List<IBindingProvider>();
+        private List<FluentBindToInputOptions> _binders = new List<FluentBindToInputOptions>();
```

Comment:
Either the type name is off here, or the member name _binders. FluentBindToInputOptions doesn't seem like a comprehensible type name :)